apps/graphql/lib/graphql/schema/shell.ex

```diff
@@ -98,7 +102,6 @@ defmodule GraphQl.Schema.Shell do
     field :network,       :network_configuration
     field :bucket_prefix, :string
     field :cluster,       :string
-    field :region,        :string
```

Comment:
what what is this?

Reply:
I just reverted this PR: https://github.com/pluralsh/plural/pull/869

Since the region is anyway null here.

Reply:
oh wait, it is actually needed there for the field resolve to work, right?

Reply:
if it's queried then yea it'll break if removed, I think we should probably just leave it for now